integrations/slack/src/router.ts

```diff
@@ -52,17 +73,24 @@ export const handleFetchEvent: FetchEventCallback = async (request, context) =>
         })
     );
 
+    // event triggers, e.g app_mention
+    router.post('/events', verifySlackRequest, acknowledgeSlackEvent);
+
+    // shortcuts & interactivity
+    router.post('/actions', verifySlackRequest, acknowledgeSlackAction);
+
+    // /gitbook
+    router.post('/commands', acknowledgeSlackCommand);
```

Comment:
I'm thinking this should look like something like this

    // event triggers, e.g app_mention
    router.post('/events', verifySlackRequest, acknowledgeSlackRequest);

    // shortcuts & interactivity
    router.post('/actions', verifySlackRequest, acknowledgeSlackRequest);

    // /gitbook
    router.post('/commands', verifySlackRequest, acknowledgeSlackRequest);

Reply:
reworked it and it's much cleaner now :)